src/main/features/ccj/form/models/repaymentPlan.ts

```diff
@@ -0,0 +1,82 @@
+import { LocalDate } from 'forms/models/localDate'
+import { PaymentSchedule } from 'ccj/form/models/paymentSchedule'
+import { IsDefined, IsIn, IsPositive, ValidateNested } from 'class-validator'
+import { IsValidYearFormat } from 'forms/validation/validators/isValidYearFormat'
+import { IsValidLocalDate } from 'forms/validation/validators/isValidLocalDate'
+import { IsFutureDate } from 'app/forms/validation/validators/dateFutureConstraint'
+import { IsLessThan } from 'forms/validation/validators/isLessThan'
+
+export class ValidationErrors {
+  static readonly FIRST_PAYMENT_AMOUNT_INVALID: string = 'Enter a valid amount of first payment'
+  static readonly INSTALMENTS_AMOUNT_INVALID: string = 'Enter a valid amount for equal instalments'
+  static readonly FUTURE_DATE: string = 'Enter a first payment date in the future'
+  static readonly INVALID_DATE: string = 'Enter a valid date of first payment'
+  static readonly SELECT_PAYMENT_SCHEDULE: string = 'Select how often they should pay'
+}
+
+export class RepaymentPlan {
+
+  remainingAmount?: number
```

Comment:
no validation at all?

Reply:
its not entered by the user, its used in validation